kwikapi/api.py

```diff
@@ -455,8 +455,13 @@ def _find_request_protocol(self, request):
         return self.PROTOCOLS[protocol]
 
     def _handle_exception(self, req, e):
-        message = e.message if hasattr(e, 'message') else str(e)
-        message = '[(%s) %s: %s]' % (self.api._id, e.__class__.__name__, message)
+        if len(e.args) == 1 and type(e.args[0]) is dict:
```

Comment:
@pavan-maddula How will the service level code use this. Please post here an example of how an exception is raised in a service API method.

Reply:
Example:
raising error with a dictionary as a parameter in service level.
```
try:
        ...
except:
        raise Exception({'message':self.error_message, 'code':103})
```

Reply:
@pavan-maddula @mohansriharsha Please call me about this